css/at-rules/supports.json

```diff
@@ -80,13 +80,13 @@
             "description": "<code>selector()</code>",
             "support": {
               "chrome": {
-                "version_added": "82"
+                "version_added": false
```

Comment:
This should be `83`, given that it’s shipping: https://crbug.com/979041#c13

Reply:
Given that this PR is already merged, please open a new pull request for these changes.  :)

Reply:
I had already done so in: <https://github.com/mdn/browser-compat-data/pull/5927>.